testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/auditlog/AbstractLogFieldsOfLogTestCase.java

```diff
@@ -101,15 +101,19 @@ void configureUser(final ModelControllerClient client, final CompositeOperationB
 
     void resetUser(final CompositeOperationBuilder compositeOp) {
         if (elytronEnabled) {
-            ModelNode op = Operations.createOperation("map-remove", userAuthAddress);
+            ModelNode op = Operations.createOperation("map-clear", userAuthAddress);
             op.get("name").set("properties");
-            op.get("key").set(DEFAULT_USER_KEY);
             compositeOp.addStep(op.clone());
             op = Operations.createOperation("map-put", userAuthAddress);
             op.get("name").set("properties");
             op.get("key").set(DEFAULT_USER_KEY);
             op.get("value").set("$local");
             compositeOp.addStep(op.clone());
+            op = Operations.createOperation("map-put", userAuthAddress);
+            op.get("name").set("properties");
+            op.get("key").set("wildfly.sasl.local-user.challenge-path");
```

Comment:
what's the relation with the YAML configuration?

Reply:
to reset the configuration